src/System.ComponentModel.Primitives/src/System/ComponentModel/DescriptionAttribute.cs

```diff
@@ -52,17 +45,11 @@ public override bool Equals(object obj)
             }
 
             DescriptionAttribute other = obj as DescriptionAttribute;
-            return (other != null) && other.Description == Description;
+            return other?.Description == Description;
```

Comment:
Ouch you're right, sorry, was in robot cleanup mode and forgot string is a reference type. I've fixed this and all the others and added tests